stdlib/task/src/main/ballerina/src/task/scheduler.bal

```diff
@@ -87,7 +87,7 @@ public type Scheduler object {
 
     # Resumes a paused task.
     #
-    # + return - Returns `task:SchedulerError` when an error occurred while pausing, nil Otherwise.
+    # + return - `task:SchedulerError` when an error occurred while resuming, nil otherwise
```

Comment:
```suggestion
    # + return - `task:SchedulerError` when an error occurred while resuming, nil otherwise
```
Same as above. Also, please refer the template for the correct format of the return statement description.